components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/step/impl/DefaultStepHandler.java

```diff
@@ -104,6 +105,26 @@ public void handle(HttpServletRequest request, HttpServletResponse response,
 
             stepConfig.setCompleted(true);
             return;
+        } else {
+            long authTime = 0;
+            if (StringUtils.isNotEmpty(request.getParameter(FrameworkConstants.RequestParams.MAX_AGE)) && StringUtils.
```

Comment:
request.getParameter(FrameworkConstants.RequestParams.MAX_AGE) assign to variable